supportedBackends/bcs/src/main/scala/cromwell/backend/impl/bcs/BcsConfiguration.scala

```diff
@@ -44,6 +44,7 @@ final class BcsConfiguration(val configurationDescriptor: BackendConfigurationDe
   lazy val dockerHashAccessId = DockerConfiguration.dockerHashLookupConfig.as[Option[String]]("alibabacloudcr.auth.access-id")
   lazy val dockerHashAccessKey = DockerConfiguration.dockerHashLookupConfig.as[Option[String]]("alibabacloudcr.auth.access-key")
   lazy val dockerHashSecurityToken = DockerConfiguration.dockerHashLookupConfig.as[Option[String]]("alibabacloudcr.auth.security-token")
+  lazy val dockerHashEndpoint: Option[String] = DockerConfiguration.dockerHashLookupConfig.as[Option[String]]("alibabacloudcr.auth.endpoint")
```

Comment:
This is consistent with the previous lines, so no need to change this line. For those looking for a slightly shorter Option...

```scala
lazy val dockerHashEndpoint: Option[String] = DockerConfiguration.dockerHashLookupConfig.getAs[String]("alibabacloudcr.auth.endpoint")
```

Reply:
Done